tests/conftest.py

```diff
@@ -59,6 +59,11 @@ def firefox_options(firefox_options):
     return firefox_options
 
 
+@pytest.fixture(scope="session")
+def browser_context_args(browser_context_args):
+    return {**browser_context_args, "ignore_https_errors": True}
```

Comment:
Why is this needed?

Reply:
ignores SSL errors and certificate warnings